.github/workflows/build.yml

```diff
@@ -48,6 +48,13 @@ jobs:
       env:
         VERSION: ${{ steps.version.outputs.result }}
         GRADLE_BUILD_ACTION_CACHE_DEBUG_ENABLED: true
+    - name: Game Test
+      uses: gradle/gradle-build-action@v2
+      with:
+        arguments: runGameTestServer
+      env:
+        VERSION: indev-test
+        GRADLE_BUILD_ACTION_CACHE_DEBUG_ENABLED: true
```

Comment:
Can you not append arguments to the previous action? This looks like it will repeat the build which would be a waste.